src/Debugging-Core/Context.extension.st

```diff
@@ -22,15 +22,13 @@ Context >> callPrimitive: primNumber [
 { #category : #'*Debugging-Core' }
 Context >> lookupSymbol: aSymbol [
```

Comment:
I think we should rename this method to something like #valueOfSymbol:. I can do it in another PR. Suggestions for name are welcome.

Reply:
yes, the name is not good... maybe valueForVariableNamed? valueForSymbol:?

Reply:
and we need another name to modify the value. Basically they will be generalized version of tempNamed:/tempNamed:put:

Reply:
I think #read/#write are good words to use there:
- aContext readVarNamed: aSymbol
- aContext write: aValue toVarNamed: aSymbol